repository/mock/feeds.go

```diff
@@ -21,3 +24,28 @@ func GetFeeds() (*feeds.Response, error) {
 		Results: response,
 	}, nil
 }
+
+func GetFeedDetail(neighborhoodId int) (*feeds.FeedDetailResponse, error) {
+	var response feeds.FeedDetailResponse
+	for _, nh := range repository.NeighborhoodIdToMap {
+		if nh.Id == neighborhoodId {
+			response.NeighborhoodId = nh.Id
+			response.Intensity = rand.Intn(5) + 1
+			response.LastUpdateTime = time.Now().Add(-time.Minute * time.Duration(rand.Intn(60))).Format(time.RFC3339)
+			response.Details = make([]feeds.FeedDetail, 0)
+			for i := 0; i < rand.Intn(10)+1; i++ {
+				response.Details = append(response.Details, feeds.FeedDetail{
+					BuildingName: fmt.Sprintf("Building %d", i),
+					BallotBoxNos: []int{
+						i - 1,
```

Comment:
The ballot box number shouldn't contain a negative value in the mock data. Mock data and the prod data should be consistent